rules/review-videos-collaboratively/rule.md

```diff
@@ -28,9 +27,12 @@ So you have just finished that sweet video edit and you are now ready & excited
 
 This is where collaborative viewing platforms like [Frame.io](https://frame.io) come in. 
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/rapbhpW3jPM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
Hey @williamgreentreessw 

1. Please see https://ssw.com.au/rules/rule/ and follow the right markdown to add a video

Reply:
Done, followed rule to embed video